frame/staking/src/remote_ext_tests.rs

```diff
@@ -0,0 +1,75 @@
+use remote_externalities::{Builder, Mode, OnlineConfig};
```

Comment:
Right now I am planning on moving these tests to the polkadot repo so they are concrete over the exact runtime types.

Polkadot branch name: zeke-companion-for-9081-tests

Reply:
Will need to remove the new deps in Cargo.toml as well